package.json

```
@@ -99,7 +99,7 @@
     "jest-jasmine": "JEST_JASMINE=1 yarn jest",
     "jest-jasmine-ci": "yarn jest-jasmine --color --config jest.config.ci.js",
     "jest-coverage": "yarn jest --coverage",
-    "lint": "eslint . --cache --ext js,jsx,ts,tsx,md",
+    "lint": "eslint . --cache --ext js,jsx,mjs,ts,tsx,md",
```

Nice. Shall we add `cjs` as well? There are few of them in e2e tests.

```suggestion
    "lint": "eslint . --cache --ext js,jsx,cjs,mjs,ts,tsx,md",
```